### ISSUE TITLE
completed-docs reports missing docs on interfaces even though interfaces is disabled

### ISSUE THREAD

**Issue by 0815fox** [palantir/tslint#4493]
### Bug Report

- __TSLint version__: 5.12.1 (last working version: 5.11.0)
- __TypeScript version__: 3.1.6
- __Running TSLint via__: CLI

#### TypeScript code being linted

```ts
/**
 * Interface used to identify a node.
 */
export interface INode {
	getDeviceId(forceRead?: boolean): Promise<string>;
}
```

with `tslint.json` configuration:

```json
"completed-docs": true
```
but also fails with
```json
"completed-docs": [true, "classes", "functions", "methods", "properties"],
```

#### Actual behavior
```
ERROR: /home/mkr/work/albula/src/Node.ts:42:2 - Documentation must exist for methods.
```
#### Expected behavior
completed-docs should not report missing documentation on interfaces unless enabled (https://palantir.github.io/tslint/rules/completed-docs/)

May be a regression from #3534

**Comment by jeanlucburot** [comment]
Any progress on this one? I'm having exactly this problem with `"jsonschema": "1.2.5"`.

**Comment by JoshuaKGoldberg** [comment]
### 💀 _**It's time!**_ 💀

**TSLint is deprecated** and no longer accepting pull requests other than security fixes. See #4534. ☠️ 
We recommend you instead use **[typescript-eslint](https://typescript-eslint.io)** to lint your TypeScript code with **ESLint**. ✅

👋 It was a pleasure open sourcing with you!

**Comment by JoshuaKGoldberg** [comment]
🤖 Beep boop! 👉  **TSLint is deprecated** 👈 _(#4534)_  and you should **switch to [typescript-eslint](https://typescript-eslint.io)**! 🤖 

🔒 This issue is being locked to prevent further unnecessary discussions. Thank you! 👋
